lib/opal/nodes/rescue.rb

```diff
@@ -10,6 +10,9 @@ class EnsureNode < Base
       children :begn, :ensr
 
       def compile
+        # FIXME: We don't need this if there's no rescue else
+        push_closure
```

Comment:
push_closure if has_rescue_else?

maybe

Reply:
unfortunately no... has_rescue_else? can be called only after we have traversed thru all the child nodes. as a followup to this, I plan to mark the throwers using a rewriter, replacing the current measures and the BreakFinder rewriter